floris/simulation/flow_field.py

```diff
@@ -150,6 +150,12 @@ def finalize(self, unsorted_indices):
         self.w = np.take_along_axis(self.w_sorted, unsorted_indices, axis=2)
 
     def calculate_speed_ups(self, het_map, x, y, z=None):
+
+        if x.shape[0] != np.array(het_map).shape[1]: # Check that the het maps wd dimension matches
```

Comment:
Consider user `self.n_wind_directions` here rather than `x.shape[0]` since it is more explicit.

Reply:
Done!